charts/rstudio-connect/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: rstudio-connect
 description: Official Helm chart for RStudio Connect
-version: 0.2.39-alpha03
```

Comment:
based on previous helm chart examples, I would have assumed this went to `0.2.40`. How do we know what to change it to?
I'd like to add more detail to the wiki page to better document!

Reply:
Yeah, normally 0.2.40 would have been the right call 😄 This release is special because it is the launching of the trebuchet beta. You can leave the docs at suggesting 0.2.40 😄 

(Basically we "already" landed 2022.07 in a context that is "hidden." This leaves it hidden until we finish testing a bunch of chart changes on Colorado)